storops_test/lib/test_common.py

```diff
@@ -357,3 +358,23 @@ def test_user_cap_no(self):
         size_byte = bitmath.GiB(3).to_Byte().value
         new_size = supplement_filesystem(size_byte, False)
         assert_that(new_size, equal_to(bitmath.GiB(3).to_Byte().value))
+
+
+class InitiatorFormatTest(TestCase):
+    def test_iscsi_format(self):
+        assert_that(common.is_iscsi_uid('iqn.1994-05.com.redhat:4dc0b7e1edc9'),
```

Comment:
Please separate each `assert_that` into their own test cases with a test case name indicating the purpose of that assert.  The benefit is that even if one of them failed we will still run test for the others.  And it's easier to root cause the failure.